src/flux/performance-export.coffee

```diff
@@ -0,0 +1,85 @@
+{CrudConfig, makeSimpleStore, extendConfig} = require './helpers'
+{JobActions, JobStore} = require './job'
+_ = require 'underscore'
+moment = require 'moment'
+
+EXPORTING = 'exporting'
+EXPORT_QUEUED = 'export_queued'
+EXPORTED = 'completed'
+
+PerformanceExportConfig = {
+
+  _job: {}
+
+  _loaded: (obj, id) ->
+    @emit('performanceExport.loaded', id)
+
+  _updateExportStatusFor: (id) ->
+    (jobData) =>
+      exportData = _.clone(jobData)
+      exportData.exportFor = id
+
+      @_asyncStatus[id] = exportData.state
+      @emit("performanceExport.#{exportData.state}", exportData)
+      @emitChange()
+
+  getJobIdFromJobUrl: (jobUrl) ->
+    jobUrlSegments = jobUrl.split('/api/jobs/')
```

Comment:
@joemsak  could the `POST` which returns the Job URL return the job id instead? That way Amanda does not have to parse the `id` from the URL